src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+  static alive = [];
+
+  static animalHealth(animal) {
```

Comment:
```suggestion
  static checkHealth(animal) {
```
Function/method name should start from verb